statistic-labels/index.html

```diff
@@ -327,6 +327,7 @@ <h3 id="printBasinTypeHeader">Basin Characteristic Types</h3>
 
 			<p class="showStatTypes">
 				The values below are used in StreamStats as statistic labels for at-site statistics for streamgages and regression-based scenarios at user-selected ungaged locations.
+				Not all statistic labels are available in certain states.
```

Comment:
I would change to something like "Not all statistic labels are available in all states. Use the "Filter by Region" drop-down box to view the statistic labels available for a specific state."